multi-agent/lib/mas/statistics/service.py

```
@@ -333,7 +334,17 @@ def _build_user_activity_list(
             activity.blueprints_used = len(identity_blueprints.get(key, set()))
         
-        result = sorted(user_data.values(), key=lambda x: x.run_count, reverse=True)
+        def _actual_runs(activity: UserActivity) -> int:
+            return sum(
+                c for s, c in activity.status_breakdown.items()
+                if s not in NON_RUNNABLE_STATUSES
+            )
+
+        result = sorted(
+            (a for a in user_data.values() if _actual_runs(a) > 0),
+            key=lambda x: _actual_runs(x),
```

Ruff (0.15.15)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)